src/func-cosmosdb-worker/src/functions/xxENSONOxx.xxSTACKSxx.Worker/CosmosDbChangeFeedEvent.cs

```diff
@@ -1,25 +1,36 @@
-using Newtonsoft.Json;
+//using Newtonsoft.Json;
```

Comment:
Comment? 🙂

Reply:
Thanks for the comment.
Still a work in progress a moment, just wanted to check the builds pass before finalising. 🤞